main.tf

```diff
@@ -209,12 +209,14 @@ data "template_file" "runners" {
     docker_machine_options      = length(var.docker_machine_options) == 0 ? "" : local.docker_machine_options_string
     runners_name                = var.runners_name
     runners_tags = var.overrides["name_docker_machine_runners"] == "" ? format(
-      "%s,Name,%s-docker-machine",
+      "%s,%s,Name,%s-docker-machine",
       local.tags_string,
+      local.runner_tags_string,
       var.environment,
       ) : format(
-      "%s,Name,%s",
+      "%s,%s,Name,%s",
       local.tags_string,
+      local.runner_tags_string,
       var.overrides["name_docker_machine_runners"],
```

Comment:
I think we have to change the order of the tags added to the string. Such that in the default case  (case name_docker_machine_runners is empty) as follow:

First add the default Name to string, next the tags_string and finally the runner_tags_astring. In this case the runner_tags string overwrite all the previous and so on. 

The code will be than the follow:

```      "Name,%s-docker-machine,%s,%s",
      var.environment,
      local.tags_string,
      local.runner_tags_string,
      ) : format(
      "%s,%s,Name,%s",
      local.tags_string,
      local.runner_tags_string,
      var.overrides["name_docker_machine_runners"],```

Reply:
I have changed the order, but not sure if I understand, please check the change.

Reply:
thanks, will check as soon as possible.